package.json

```diff
@@ -53,9 +53,8 @@
         "typedoc-plugin-missing-exports": "^3.0.0",
         "typescript": "^5.6.3"
     },
-    "engineStrict": true,
     "engines": {
-        "node": ">=14.0.0",
+        "node": ">=20.18.0",
```

Comment:
Note for future visitors. At the time this was landed, this was two LTS versions ago.